lib/byebug/helpers/toggle.rb

```diff
@@ -26,14 +24,22 @@ def enable_disable_breakpoints(is_enable, args)
           selected_breakpoints = all_breakpoints.select do |b|
             selected_ids.include?(b.id)
           end
+          return selected_breakpoints
```

Comment:
Don't use explicit `return` unless necessary. Also, after this refactoring the `selected_breakpoints` variable is no longer necessary.